DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/RevitConversionSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Speckle.Converters.RevitShared;
+
+// POC: probably NOT the right place, probably needs passing in with the send/rcv operation
+// not clear how this should get configured or if we should have it, the shape probably needs to change
+// this was dragged in because it (or something like it) is required for reference point conversion.
+// have made it a strongly typed thing encapsulating a dictionary so as not to rely on injecting a weak type.
```

Comment:
If I recall correctly. The setting you're referring to allowed users to select which origin point they want to use as the 0,0 coordinates for sending.
If we're not supporting this as of now we can just safely ignore.

I think adding settings to the converter was kindof a bad idea. My thought is that the Context should be replacing the settings in most of the cases we were using it.

Reply:
I mean I can take it out, I was just bringing it across, I can assume it's null or string.Empty and that will simplify things a bit

Reply:
Added a comment around reviewing this reference point and the settings